test/functional/data.spec.js

```diff
@@ -23,6 +25,12 @@ function checkURL(url, head = false) {
 }
 
 describe('swag-for-dev', function () {
+	it('data.json has valid schema', function () {
+		const ajv = new Ajv();
+		const valid = ajv.validate(schema, data);
+		expect(valid).to.be.true;
+	});
+
 	it('data.json is valid', function () {
```

Comment:
This test will be replaced by 'data.json has valid schema'.

Reply:
I think this should be merged with the schema test since it checks that everything is in alphabetical order

Reply:
`chai-json-schema-ajv` plug-in should be a good fit for running tests.

Reply:
No need to add an extra dep 😄 what you have now for validation is good / on the right track 👍🏾